### DISCUSSION TITLE
Image as custom shape for scatter dots

### DISCUSSION THREAD

**Discussion by jeadys** [recharts/recharts#3928] [Q&A] [answered]
How can I use an image instead of a colorful dot as a scatter point?

I tried something like below but this doesn't work.

``` js
  const CustomScatterShape = () => {
    return (
      <Image src="/images/scatter.webp" alt="scatter" width="0" height="0" sizes="100vw" className="h-10 w-10 rounded-md object-cover" />
    )
  }
  
  <Scatter data={trackAudioFeatures} fill="#93c5fd" shape={<CustomScatterShape />} />

```

**Comment by jeadys** [comment]
An example here of images used in a graph

![D5yKa](https://github.com/recharts/recharts/assets/107939865/7e8cb9bf-8ac4-476d-bcbe-120175e06fa4)

**Comment by ckifer** [comment]
Hi, what component are you using for Image?

**Reply by jeadys** [reply]
I'm using [Next Image](https://nextjs.org/docs/pages/api-reference/components/image) component. I also tried using standard HTML img tag.

**Reply by ckifer** [reply]
SVGs require the `<image>` tag rather than img. Can you try that?

**Reply by jeadys** [reply, marked as answer]
This works indeed, thanks for the help. Here is the result
``` js

// cx and cy for correct coordinates
const CustomScatterShape = ({ cx, cy }: DotProps) => {
  return (
    <svg width="50" height="50" x={cx} y={cy}>
      <image href="/images/dummy.webp" width="50" height="50" />
    </svg>
  )
}

// Passing CustomScatterShape in shape property
<Scatter data={trackAudioFeatures} shape={<CustomScatterShape />} />
```

![scatter](https://github.com/recharts/recharts/assets/107939865/0c227d13-51e6-4dd6-85e4-21d7d6b664a0)

**Reply by ckifer** [reply]
Nice, glad to hear it!
